### PR TITLE
Add name and username to README (Siddharth Kannan)

### PR DESCRIPTION
**Pull request by icyflame**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,5 @@ Lines starting with `*` are automatically rendered as bullet points and phrases
 * Harsh Gupta [hargup](http://github.com/hargup)
 * Krishna Bagadia [krishna95](http://github.com/krishna95)
 * Nevin Valsaraj [routeaccess](http://github.com/routeaccess)
+* Siddharth Kannan [icyflame](http://github.com/icyflame)
+
```
Comment (hargup) [inline README.md]: Remove the redundant newline.

**Comment by icyflame** [comment]
@hargup Edited as required. 

I added the newline as adding a newline at the end of a file seemed conventional.

**Comment by hargup** [comment]
@icyflame under normal circumstances your editor automatically adds a newline at the end of each line, even if that line is last line of the file. Though it possible to create files without an eol at the end of last line and the diff of github highlights that. I created such a file for demostration see https://github.com/hargup/git-sandbox/pull/1/files

**Comment by hargup** [comment]
:tada:

**Pull request merged**